docs/minio-job.md

```diff
@@ -0,0 +1,209 @@
+# MinioJob is a Kubernetes Job that runs mc commands
+here is an example of a MinioJob:
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: mc-job-sa
+---
+apiVersion: sts.min.io/v1alpha1
+kind: PolicyBinding
+metadata:
+  name: mc-job-binding
+spec:
+  application:
+    serviceaccount: mc-job-sa
+  policies:
+    - consoleAdmin
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mytestsecret
+data:
+  PASSWORD: cGVkcm8xMjM=
```

Comment:
We may want to add a comment that the password is set to `pedro123`. Every K8s guy should know that it's base64, but it adds to the readability of the example.

Reply:
I expect this framework to be used by customers with low K8s familiarity because of it's inherent power / flexibility, so we should remind people anyways that these payloads are all encoded.